### DISCUSSION TITLE
ipv6 ? 🤷‍♂️ upd: Vercel shuts down such issues due to disregard for its users. 👎

### DISCUSSION THREAD

**Discussion by Pilaton** [vercel/vercel#10626] [Ideas]
### Idea

Isn't it time to introduce ipv6 support already? 
The main ipv4 is blacklisted by many providers in some countries and sites located on it are blocked. 

If there are difficulties with the implementation of ipv6, maybe then expand the list of ipv4 addresses?

**Comment by leerob** [comment]
Duplicate of https://github.com/orgs/vercel/discussions/47
